tests/functional/amdApp/circular2.ts

```diff
@@ -0,0 +1,9 @@
+import * as circular1 from './circular1';
```

Comment:
You can change this to:
`import { getMessage } from './circular1';`

And them below replace `circular1.getMessage()` with `getMessage()`.